src/sentry/api/endpoints/organization_code_mapping_details.py

```diff
@@ -5,7 +5,10 @@
 from rest_framework.response import Response
 
 from sentry.api.base import region_silo_endpoint
-from sentry.api.bases.organization import OrganizationEndpoint, OrganizationIntegrationsPermission
```

Comment:
This grants any organization _member_ to read, write, update and delete the _details_ of code mappings. Is this correct?

Reply:
that's the goal i believe